co-circom/co-circom/src/bin/co-circom.rs

```diff
@@ -477,7 +477,10 @@ where
 
     let public_input = match proof_system {
         ProofSystem::Groth16 => {
-            let zkey = Arc::new(Groth16ZKey::<P>::from_reader(zkey_file).context("reading zkey")?);
+            let zkey = Arc::new(
+                Groth16ZKey::<P>::from_reader(zkey_file, CheckElement::Yes)
```

Comment:
this should probably be configurable from env

Reply:
we agreed to move it to later